### ISSUE TITLE
Version 1.4.0 breaks when extra contains is null or undefined

### ISSUE THREAD

**Issue by weaintplastic** [marcohamersma/react-bem-helper#10]
I'm using react-bem-helper passing down the className from props as an extra like this:

```
bem({
      modifiers: {...},
      extra: [
        status,
        className,
      ],
    })
```

Since version 1.4.0 it breaks when className is null/undefined with the following error message:

```
TypeError: Cannot read property 'split' of undefined
      at stringToArray (node_modules/react-bem-helper/index.js:21:16)
      at node_modules/react-bem-helper/index.js:48:27
      at Array.reduce (native)
      at listToArray (node_modules/react-bem-helper/index.js:47:17)
      at node_modules/react-bem-helper/index.js:102:29
```

**Comment by marcohamersma** [comment]
This should be fixed now in 1.4.1

**Comment by weaintplastic** [comment]
thanks a lot!
